examples/notebooks/shell.nix

```diff
@@ -0,0 +1,13 @@
+# brings poetry into scope and makes sure that poetry-installed C-lib basedpython
```

Comment:
@tobim : maybe you want to have a brief look at this rudimentary `shell.nix`? I know it's not pretty, but at least allows me to run the docs locally.

Reply:
The essential thing is that it does the job. We can worry about "beauty" some other time.